hawkbit-mgmt-resource/src/test/java/org/eclipse/hawkbit/mgmt/rest/resource/MgmtDistributionSetTypeResourceTest.java

```diff
@@ -554,7 +555,7 @@ public void invalidRequestsOnDistributionSetTypesResource() throws Exception {
                 .contentType(MediaType.APPLICATION_JSON)).andDo(MockMvcResultPrinter.print())
                 .andExpect(status().isBadRequest());
 
-        final DistributionSetType missingName = entityFactory.generateDistributionSetType("test123", null, "Desc123");
+        final DistributionSetType missingName = new JpaDistributionSetType("test123", null, "Desc123");
```

Comment:
why using here `new JpaDistributionSetType` and not like before the `entityFactory.generateDistributionSetType`